config/unicorn.conf.rb

```diff
@@ -79,19 +79,16 @@
       end
     end
 
+    # Initialize the SignalTrapLogger singleton and start logging thread
+    SignalTrapLogger.instance
+
     sidekiqs = ENV["UNICORN_SIDEKIQS"].to_i
     if sidekiqs > 0
       server.logger.info "starting #{sidekiqs} supervised sidekiqs"
 
       require "demon/sidekiq"
-      Demon::Sidekiq.logger = server.logger
       Demon::Sidekiq.after_fork { DiscourseEvent.trigger(:sidekiq_fork_started) }
-      Demon::Sidekiq.start(sidekiqs)
-
-      Signal.trap("SIGTSTP") do
-        Demon::Sidekiq.log("Issuing stop to Sidekiq")
-        Demon::Sidekiq.stop
-      end
```

Comment:
Note that I'm removing this for two reasons:

1. When `DISCOURSE_ENABLE_EMAIL_SYNC_DEMON` == `true`, it will break this because it traps the `SIGTSTP` signal as well. 

2. There is almost no scenario where we need to manually stop sidekiq? Even if Sidekiq is stopped, our heartbeat check will start it again so what is the point of this?